src/System.Collections.Immutable/tests/project.lock.json

```diff
@@ -2,61 +2,175 @@
   "locked": true,
   "version": -9997,
   "targets": {
-    ".NETPortable,Version=v4.5,Profile=Profile259": {
-      "xunit/2.0.0-beta5-build2785": {
+    "DNXCore,Version=v5.0": {
+      "System.Collections/4.0.10-beta-22816": {
         "dependencies": {
-          "xunit.core": "[2.0.0-beta5-build2785]",
-          "xunit.assert": "[2.0.0-beta5-build2785]"
-        }
+          "System.Runtime": "4.0.20-beta-22816"
+        },
+        "compile": [
+          "lib/contract/System.Collections.dll"
+        ],
+        "runtime": [
+          "lib/aspnetcore50/System.Collections.dll"
+        ]
       },
-      "xunit.abstractions/2.0.0-beta5-build2785": {
+      "System.Diagnostics.Contracts/4.0.0-beta-23008": {
```

Comment:
So what should I do here?  Regenerate both the src and tests project.lock.json files at the same time?  These edits to the lock file were all automated from my changing the project.json.